### ISSUE TITLE
Create Homebrew Cask formula

### ISSUE THREAD

**Issue by prawel** [psugihara/FreeChat#62]
It would be nice to be able to install this without logging in to Apple Store or building using XCode. Homebrew Cask formula seems like a good solution.

**Comment by psugihara** [comment]
yeah that would be cool. curious, would it be better to just have a signed release on freechat.run you could download?

**Comment by prawel** [comment]
both options solve the problem. Personally I prefer using homebrew, but compiled binary is good enough
